recipes/recipes_emscripten/sparrow/recipe.yaml

```diff
@@ -19,8 +19,6 @@ requirements:
   - make
   host:
   - howardhinnant_date
-  run:
-  - howardhinnant_date
```

Comment:
The latest commit addresses this 

https://github.com/emscripten-forge/recipes/pull/1847#discussion_r1938836086